_studio/shared/umc/core/vm/src/vm_semaphore_linux32.c

```diff
@@ -69,44 +70,55 @@ vm_status vm_semaphore_init(vm_semaphore *sem, int32_t init_count)
 
     sem->count = init_count;
     sem->max_count = 1;
-    res = pthread_cond_init(&sem->cond, 0);
+    res = pthread_condattr_init(&cond_attr);
     if (!res)
     {
-        res = pthread_mutex_init(&sem->mutex, 0);
-        if (res)
+        res = pthread_cond_init(&sem->cond, 0);
```

Comment:
And here?

Reply:
Done. I missed this when I ported from my test environment.

Reply:
Yes indeed. I missed this when I ported from my test environment (Android).